pybamm/parameters/lithium_ion_parameters.py

```diff
@@ -255,14 +255,14 @@ def _set_dimensional_parameters(self):
         self.m_cr_n = pybamm.Parameter("Negative electrode Paris' law constant m")
         self.Eac_cr_n = pybamm.Parameter(
             "Negative electrode activation energy for cracking rate [kJ.mol-1]"
-        )  # noqa
+        )  # noqaR
```

Comment:
should this R be here?

Reply:
No, that is a typo I introduced (how ironic haha)